active/AnimatedIcon/AnimatedIconSpec.md

```diff
@@ -0,0 +1,894 @@
+* PointerOverOnToNormalOn
+* PointerOverOnToPressedOn
+* PointerOverOffToPointerOverOn
+* PointerOverOffToNormalOff
+* PointerOverOffToPressedOff
+* PressedOnToPressedOff
+* PressedOnToPointerOverOff
+* PressedOnToNormalOff_Start
+* PressedOffToPressedOn
+* PressedOffToPointerOver
+* PressedOffToNormalOn
+
+## AnimatedGlobalNavigationButtonVisualSource class
+
+This class can be used to show an animation for a Navigation operation.
+This can be used as the Source of an AnimatedIcon and is used by the XAML NavigationView control.
+
+This class supports markers that can be driven by an AnimatedIcon.State:
+* TBD
+
+## AnimatedBackVisualSource class
+
+This class can be used to show an animation for a Navigate-Back operation.
+This can be used as the Source of an AnimatedIcon and is used by the XAML NavigationView control.
+
+This class supports markers that can be driven by an AnimatedIcon.State:
+* TBD
+
+## AnimatedSettingsVisualSource class
+
+This class can be used to show an animation for a Settings icon.
+This can be used as the Source of an AnimatedIcon and is used by the XAML NavigationView control.
+
+This class supports markers that can be driven by an AnimatedIcon.State:
+* TBD
+
+## AnimatedFindVisualSource class
+
+This class can be used to show an animation for a Find icon.
+This can be used as the Source of an AnimatedIcon and is used by the XAML AutoSuggestBox control.
+
+## AnimatedChevronDownSmallVisualSource class
+
+This class can be used to show an animation for button that opens a flyout on another button.
+This can be used as the Source of an AnimatedIcon and is used by the XAML DropDownButton and Expander controls.
+
+This class supports markers that can be driven by an AnimatedIcon.State:
+* TBD
+
+# AnimatedChevronUpDownSmallVisualSource
+
+This class can be used to show an animation for button that opens a flyout on another button.
+This can be used as the Source of an AnimatedIcon and is used by the XAML NavigationView control.
+
+This class supports markers that can be driven by an AnimatedIcon.State:
+* TBD
+
+# AnimatedChevronRightDownSmallVisualSource class
+
+This class can be used to show an animation for a button that opens a flyout to the right of another button.
+This can be used as the Source of an AnimatedIcon and is used by the XAML TreeView control.
+
+This class supports markers that can be driven by an AnimatedIcon.State:
+* TBD
+
+
+
+# API Details
+
+## Microsoft.UI.Xaml.Controls
+
+```c++
+[webhosthidden]
+interface IAnimatedVisualSource2
+{
+    IAnimatedVisual TryCreateAnimatedVisual(Windows.UI.Composition.Compositor compositor);
+
+    Windows.Foundation.Collections.IMapView<String, Double> Markers { get; };
+    void SetColorProperty(String propertyName, Windows.UI.Color value);
+};
+
+[webhosthidden]
+[contentproperty("Source")]
+unsealed runtimeclass AnimatedIcon : Windows.UI.Xaml.Controls.IconElement
+{
+    AnimatedIcon();
+
+    IAnimatedVisualSource2 Source{ get; set; };
```

Comment:
Super duper nit - sometimes there's a space between the property name and the brace, sometimes there isn't.